### PR TITLE
Fix typos in comments, log messages and strings

### PR DESCRIPTION
**Pull request by alexandear**
List of fixed words:
- successful
- should
- revocable
- NuGet
- production
- pre-calculated
- failed
- truly
- because
- unknown
- package
- PowerShell
- addresses
- configuration
- maintenance
- perform
- accepted
- indices
- recursive
- uppercase
- broken
- purposes
- empty

### CODE REVIEW

**Comment by skofman1** [comment]
LGTM. @scottbommarito , want to add another pair of eyez?

**Comment by shishirx34** [comment]
Looks good.

File: tests/NuGetGallery.Facts/Controllers/ApiControllerFacts.cs [hunk 1]
```diff
@@ -581,7 +581,7 @@ public class TheGetPackageAction
                 var package = new Package();
                 var actionResult = new EmptyResult();
                 var controller = new TestableApiController(MockBehavior.Strict);
-                controller.MockPackageService.Setup(x => x.FindPackageByIdAndVersion("Baz", "", false)).Throws(new DataException("Oh noes, database broked!"));
+                controller.MockPackageService.Setup(x => x.FindPackageByIdAndVersion("Baz", "", false)).Throws(new DataException("Oh noes, database broken!"));
```
Comment (scottbommarito) [inline tests/NuGetGallery.Facts/Controllers/ApiControllerFacts.cs]: I think this was actually meant to be an intentionally misspelled funny message LOL. I kinda like how it is but it's ok if you want to change it. Also, `noes` is spelled wrong as well.

Comment (alexandear) [inline tests/NuGetGallery.Facts/Controllers/ApiControllerFacts.cs]: I can revert my change for this line.

**Pull request merged**